src/Sulu/Bundle/MediaBundle/DependencyInjection/SuluMediaExtension.php

```diff
@@ -335,4 +352,28 @@ private function configureStorage(array $config, ContainerBuilder $container, Lo
 
         $container->setAlias('sulu_media.storage', 'sulu_media.storage.' . $storage)->setPublic(true);
     }
+
+    private function getSupportedMimeTypes($ghostScriptPath, $ffmpegBinary, $ffprobeBinary)
+    {
+        $mimeTypes = ['image/*'];
+
+        if ($ffmpegBinary
+            && $ffprobeBinary
+            && $this->checkCommandAvailability($ffmpegBinary)
+            && $this->checkCommandAvailability($ffprobeBinary)
+        ) {
+            $mimeTypes[] = 'video/*';
+        }
+
+        if ($ghostScriptPath && $this->checkCommandAvailability($ghostScriptPath)) {
+            $mimeTypes[] = 'application/pdf';
+        }
+
+        return $mimeTypes;
+    }
+
+    private function checkCommandAvailability($command)
+    {
+        return null !== $this->executableFinder->find($command) || is_executable($command);
```

Comment:
@alexander-schranz WDYT about this solution? The one with `basename` and `dirname` you suggested has the problem that it can also find stuff, that does not really exists, e.g. `/some/stupid/not/existing/path/gs` would be found, because `gs` is in the PATH.

Reply:
Sounds good for me but would silence the `is_executable` like its done in the executableFinder with `@is_executable` to ignore a warning about open_basedir restriction effect when open_basedir is configured.

e.g.:

```
 php -dopen_basedir=~/Documents -r "var_dump(is_executable('/tmp/world'));
```

vs

```
 php -dopen_basedir=~/Documents -r "var_dump(@is_executable('/tmp/world'));
```

Reply:
Fixed